README.md

```diff
@@ -44,22 +44,22 @@ $ brew remove eosio
 
 #### Ubuntu 18.04 Package Install
 ```sh
-$ wget https://github.com/eosio/eos/releases/download/v1.8.0/eosio_1.8.0-1-ubuntu-18.04_amd64.deb
-$ sudo apt install ./eosio_1.8.0-1-ubuntu-18.04_amd64.deb
+$ wget https://github.com/eosio/eos/releases/download/v1.8.1/eosio_1.8.1-1-ubuntu-18.04_amd64.deb
+$ sudo apt install ./eosio_1.8.1-1-ubuntu-18.04_amd64.deb
```

Comment:
This is not relevant to our pipeline work, so let's make this a separate PR to `release/1.8.x`.